src/node_sea.cc

```diff
@@ -161,6 +205,58 @@ void IsExperimentalSeaWarningNeeded(const FunctionCallbackInfo<Value>& args) {
       sea_resource.flags & SeaFlags::kDisableExperimentalSeaWarning));
 }
 
+void GetCodeCache(const FunctionCallbackInfo<Value>& args) {
+  if (!IsSingleExecutable()) {
+    return;
+  }
+
+  Isolate* isolate = args.GetIsolate();
+  HandleScope scope(isolate);
+
+  SeaResource sea_resource = FindSingleExecutableResource();
+
+  if (!static_cast<bool>(sea_resource.flags & SeaFlags::kUseCodeCache)) {
+    return;
+  }
+
+  std::shared_ptr<BackingStore> backing_store = ArrayBuffer::NewBackingStore(
+      const_cast<void*>(
+          static_cast<const void*>(sea_resource.code_cache->data())),
+      sea_resource.code_cache->length(),
+      [](void* /* data */, size_t /* length */, void* /* deleter_data */) {
+        // The code cache data blob is not freed here because it is a static
+        // blob which is not allocated by the BackingStore allocator.
+      },
+      nullptr);
+  Local<ArrayBuffer> array_buffer = ArrayBuffer::New(isolate, backing_store);
+  Local<DataView> data_view =
+      DataView::New(array_buffer, 0, array_buffer->ByteLength());
+
+  args.GetReturnValue().Set(data_view);
+}
+
+void GetCodePath(const FunctionCallbackInfo<Value>& args) {
+  if (!IsSingleExecutable()) {
+    return;
+  }
```

Comment:
If we guard this in user land this could be

```suggestion
  DCHECK(IsSingleExecutable());
```